src/content/popup/Metadata/EraInfoComponent.fixture.tsx

```diff
@@ -0,0 +1,10 @@
+import { EraInfo, EraMeta } from '../../years';
+
+import { EraInfoComponent } from './EraInfoComponent';
+
+const TEST_DATA: { meta: EraMeta; eraInfo: EraInfo } = {
+  meta: { type: 'era', era: '令和', year: 6, matchLen: 4 },
+  eraInfo: { reading: 'れいわ', start: 2019, yomi: 'Reiwa' },
+};
```

Comment:
As per [the tests in `popups.html`](https://github.com/birchill/10ten-ja-reader/blob/59292571221f539667221917f347caa858b3b337/tests/popups.html#L216-L223) it might be good to add a test for 元年 since it's a bit special.